src/MassTransit/Configuration/Configuration/ConsumeMessageFilterConfigurator.cs

```diff
@@ -19,6 +19,11 @@ public void Include(params Type[] messageTypes)
             Filter.Includes += message => Match(message, messageTypes);
         }
 
+        public void Include(Func<Type, bool> filter)
+        {
+            Filter.Includes += context => filter(context.GetType());
```

Comment:
what is `context.GetType()` - how is that the right way to get the type here? (confused)